src/js/modules/media/components/Gallery.js

```diff
@@ -0,0 +1,192 @@
+    position: "relative",
+    display: "flex",
+    "-webkit-box-orient": "vertical",
+    "-webkit-box-direction": "normal",
+    "-ms-flex-direction": "column",
+    flexDirection: "column",
+    justifyContent: "flex-end",
+    margin: 0,
+    "& img": {
+      margin: "0 auto",
+      maxHeight: "50vh",
+      maxWidth: "100%",
+    },
+  },
+  slideCaption: {
+    alignSelf: "flex-start",
+    width: "100%",
+    "& p": {
+      color: "#fff",
+      marginTop: "10px",
+      width: "100%",
+      fontFamily: "Minion Pro",
+      fontStyle: "italic",
+      fontSize: "17px",
+      textAlign: "center",
+    },
+  },
+  thumbnailImage: {
+    height: "100%",
+    width: "100%",
+  },
+  sliderFooter: {
+    display: "-webkit-box",
+    display: "-ms-flexbox",
+    "-webkit-box-align": "center",
+    "-ms-flex-align": "center",
+    alignItems: "center",
+    display: "flex",
+    "-webkit-box-orient": "horizontal",
+    "-webkit-box-direction": "normal",
+    "-ms-flex-direction": "row",
+    flexDirection: "row",
+    "-webkit-box-pack": "end",
+    "-ms-flex-pack": "end",
+    justifyContent: "flex-end",
+    marginTop: "20px",
+  },
+  index: {
+    color: "#ddd",
+    fontFamily: "Circular Std",
+  },
+  carouselControls: {
+    "-webkit-box-align": "end",
+    "-ms-flex-align": "end",
+    "align-items": "flex-end",
+    display: "-webkit-box",
+    display: "-ms-flexbox",
+    display: "flex",
+  },
+  controlButton: {
+    background: "#333",
+    height: "48px",
+    outline: "none",
+    marginLeft: "10px",
+    position: "relative",
+    width: "48px",
+    color: "#bbb",
+    fontFamily: "Circular Std",
+    fontSize: "14px",
+    fontWeight: 500,
+    lineHeight: 1,
+    border: 0,
+    borderRadius: "2px",
+    padding: "15px",
+    verticalAlign: "middle",
+  },
+};
+
+class Gallery extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      index: 0,
+    };
+  }
+  nextSlide = () => {
+    this.slider.slickNext();
+  };
+  prevSlide = () => {
+    this.slider.slickPrev();
+  };
+  render = () => {
+    const { classes, media } = this.props;
+    const settings = {
+      customPaging: i => {
+        return (
+          <a>
+            <img
+              key={i}
+              className={classes.thumbnailImage}
+              src={media[i].thumbAttachmentUrl}
+            />
+          </a>
+        );
+      },
+      afterChange: currentSlideIndex =>
+        this.setState({ index: currentSlideIndex }),
+      arrows: false,
+      dots: true,
+      dotsClass: "slick-dots slick-thumb",
+      infinite: false,
+      speed: 500,
+      slidesToShow: 1,
+      slidesToScroll: 1,
+    };
+    return (
+      <div className={classes.Gallery}>
+        <Slider
+          {...settings}
+          ref={c => (this.slider = c)}
+          className={classes.Slider}
+        >
+          {media.map(image => {
+            return (
+              <div className={classes.slideContainer} key={image.id}>
```

Comment:
Sorta feel like this should be its own component

Reply:
I shouldn't split up Gallery; the actions & callbacks will get too confusing. It'll be all over the place.